test/Microsoft.Extensions.ML.Tests/UriLoaderTests.cs

```diff
@@ -45,7 +45,7 @@ public void can_reload_model()
                         () => loaderUnderTest.GetReloadToken(),
                         () => changed.Set());
             
-            Assert.True(changed.WaitOne(1000), "UriLoader ChangeToken didn't fire before the allotted time.");
+            Assert.True(changed.WaitOne(60 * 1000), "UriLoader ChangeToken didn't fire before the allotted time.");
```

Comment:
```suggestion
            Assert.True(changed.WaitOne(AsyncTestHelper.UnexpectedTimeout), "UriLoader ChangeToken didn't fire before the allotted time.");
```

Reply:
This is GitExtensions specific helper, do you suggest we create similar helper in our code base?

---
In reply to: [373629379](https://github.com/dotnet/machinelearning/pull/4753#discussion_r373629379) [](ancestors = 373629379)

Reply:
Sigh. Yes. I'm all over the place today it seems.

Reply:
Here are two other examples of similar approaches:

https://github.com/dotnet/roslyn/blob/910a8b0f3da36d1fbd7acea3778b75268d0c6a59/src/VisualStudio/IntegrationTest/TestUtilities/Helper.cs#L15-L18
https://github.com/microsoft/vs-threading/blob/558f24c576cc620a00b20ed1fa90a5e2d13b0440/src/Microsoft.VisualStudio.Threading.Tests/TestBase.cs#L19-L29

I'd probably use the latter if I was doing something new.